### PR TITLE
Modified paths for rwt_robot_monitor

### PR DESCRIPTION
**Pull request by iory**
rwt-robot-plotter
rwt-robot-monitor

### CODE REVIEW

**Comment by iory** [comment]
Modified Css.

rwt_diagnostics_plotter - mozilla firefox_027

**Comment by k-okada** [comment]
can we put multiple graph on each row? for example you specify `/Other` and
`iwifi` and we get  a graph on next raw. If we always have one graph, I
think it is better to extend the horizontal range of the graph to fit
'Other ' to 'remove ' button.
Also not sure what is '/Other, /diagnostics_sample, /amber Sensor' in the
graph

--
◉ Kei Okada


2018-02-12 18:16 GMT+09:00 iory <notifications@github.com>:

**Comment by iory** [comment]
rwt_diagnostics_plotter - mozilla firefox_030

I extend the horizontal range of the graph to fit 'Other' to 'remove' button.

**Comment by iory** [comment]
Modified picture.

rwt_diagnostics_plotter - mozilla firefox_032

**Comment by k-okada** [comment]
@iory looks better now, last comment
1. please show rpm of fan
2. please capture window, instead of entire desktop and crop it.
https://qiita.com/yas-nyan/items/80f2db8c4bdf4c8e87b8
https://askubuntu.com/questions/220225/how-can-i-print-screen-just-one-window-and-not-my-entire-desktop

**Comment by xiaoerlaigeid** [comment]
Hi! I am applying GSoC in this project. I saw monitor don't have readme file. Do I need to add readme file in this directory? @iory

**Comment by iory** [comment]
@xiaoerlaigeid

Sorry for my late reply. 

>  Do I need to add readme file in this directory?

README file is very welcome. If you have a time, please send PR to this branch.

File: rwt_robot_monitor/www/index.html [hunk 1]
```diff
@@ -37,17 +37,17 @@ <h2>All</h2>
           </div>
         </div>
         <div class="footer-stats">
-          <p>Last message received <span id="last-message-sec">0</span> secnods ago
+          <p>Last message received <span id="last-message-sec">0</span> seconds ago
         </div>
       </div>
     </div>
-    <script type="text/javascript" src="/rwt_plot/flot/jquery.min.js"></script>
+    <script type="text/javascript" src="../rwt_utils_3rdparty/jquery/jquery.min.js"></script>
```
Comment (k-okada) [inline rwt_robot_monitor/www/index.html:44]: why this has to be `src="../rwt_utils_3rdparty/jquery/jquery.min.js"` and next line is `src="/rwt_utils_3rdparty/bootstrap/js/bootstrap.min.js"` ?

**Pull request closed without merging**